src/SFA.DAS.Tools.Servicebus.Support.Infrastructure/Services/CosmosDb/CosmosUserSessionDbContext.cs

```diff
@@ -71,5 +71,30 @@ public async Task<IEnumerable<UserSession>> GetExpiredUserSessions()
 
             return userSessions;
         }
+
+        public async Task<IEnumerable<UserSession>> GetUserSessionsAsync()
+        {
+            var sqlQuery = $"select * from c where c.type = 'session'";
+
+            var database = await _client.CreateDatabaseIfNotExistsAsync(_databaseName);
+            var container = await _cosmosInfrastructure.CreateContainer(database);
+
+            var queryDefinition = new QueryDefinition(sqlQuery);
+            var queryFeedIterator = container.GetItemQueryIterator<UserSession>(queryDefinition);
+            
+            var sessions = new List<UserSession>();
+
+            while (queryFeedIterator.HasMoreResults)
+            {
+                var currentResults = await queryFeedIterator.ReadNextAsync();
+
+                foreach (var session in currentResults)
+                {
+                    sessions.Add(session);
+                }
```

Comment:
Can AddRange be used here as you did previously in `GetMessageCountPerUser` to remain consistent?